msm/msm

```diff
@@ -166,11 +166,14 @@ cd "${mycroft_skill_folder}"
 for d in $(find "${mycroft_skill_folder}" -mindepth 1 -maxdepth 1 -type d |grep -v '.git'$ ); do
   if git -C "$d" rev-parse --git-dir > /dev/null 2>&1; then
     cd "${d}" 
+    rm .git/info/exclude
```

Comment:
Why would you remove an existing exclude file?  You just want to add *.pyc if it isn't in that file.  This is a little extra work, but if a dev has gone to the effort to add *.txt to an exclude then we don't want to wipe that out.